less/_components/quicksilver/featured-jumbotron.less

```diff
@@ -124,4 +124,24 @@
       color:@brand-primary;
     }
   }
+
+  // Example: project page search
+  .input-group {
+    .btn {
+      padding-left: 12px;
+      padding-right: 12px;
+      min-width: unset;
+    }
+    .form-control {
+      height: 39px;
+      border-top-left-radius: 5px;
+      border-top-right-radius: 0;
+      border-bottom-right-radius: 0;
+      border-bottom-left-radius: 5px;
+    }
+  }
+
+  #project-list-link {
+    color: @white;
```

Comment:
I don't think we should include an #id here. I am thinking the solution here is to simply set the default color of https://github.com/EclipseFdn/solstice-assets/blob/fc39ee526a072eedce05dc406b35491dc216b81a/less/_components/quicksilver/featured-jumbotron.less#L57 to white...

Reply:
But will that affect the other sites? 🤔

Reply:
Is there a way that I can add overwrite class and variables in /projects/ page? Like what we do for the Hugo sites? 🤔

Reply:
Yes, but I think it will make them better... All sites can override that value on a per-site basis. Our background image was lighter before and the previous color did work well but I think we might need to change it now since our background image are darker...

Reply:
So to confirm, you wanna change the default to white that will take effect on all the other sites?